### PR TITLE
c++ hello world

### PR DESCRIPTION
**Pull request by carter-bailey**
Finally had time. 
closes #13

### CODE REVIEW

File: hello-world/tacocat01/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*.swp
```
Comment (wrynegade) [inline hello-world/tacocat01/.gitignore:1]: Don't forget to include binary files and Cmake artifacts to your gitignore!
```suggestion
*.swp
CMakeCache.txt
CMakeFiles/
Makefile
cmake_install.cmake
helloworld
```

**Review by wrynegade — CHANGES_REQUESTED** [review]
Looks great! I can confirm this compiles and works as advertised on Linux :)

Please avoid including binary files, and don't forget to add Cmake artifacts to your .gitignore! In order to get the binary out of your git history, you will either have to rebase and remove any traces of `helloworld` or create a clean branch from our main branch. (note: deleting the binary file and recommitting will leave the binary file in the git history and won't work)

History corrections are a little easier to keep track of when you use your own branch (e.g. `tacocat01:tacocat01-helloworld` rather than `tacocat01:main`), but just be careful not to overwrite your work! :+1:

**Comment by carter-bailey** [comment]
got rid of the binary, don't know how that slipped in. I noticed that all the other pull requests only had one push so I was trying to keep mine clean as well. I had a c++ branch that I squashed into main and made the pull request from.

**Comment by wrynegade** [comment]
You'll notice the binary is still in the git-hash; you'll have to clean this guy up :)

https://github.com/tacocat01/pr-practice/commit/0b23cac686289e9582acc5e6665447510ce3a5d9#diff-d9b8b065e2a4066c90c41493d90f70d5d967f9f135bd8971e9f5b3a89b0b6883

> (note: deleting the binary file and recommitting will leave the binary file in the git history and won't work)

**Review by wrynegade — APPROVED** [review]
Looks great!

**Comment by wrynegade** [comment]
closes #1

**Pull request merged**